Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -133,6 +133,20 @@ bool CMainConfig::Load()
         return false;
     }
 
+    // Strip spaces from beginning and end of server name
+    const size_t snameStart = m_strServerName.find_first_not_of(" ");
+    if (snameStart != std::string::npos)
+    {
+        const size_t snameEnd = m_strServerName.find_last_not_of(" ");
+        m_strServerName = m_strServerName.substr(snameStart, snameEnd - snameStart + 1);
+    }
```

Comment:
It looks like a library function. You can add `trim` to `SString`

Reply:
this is already the simplest and optimized we can do imo

Reply:
String operations here are universal and could be reused in another code.
You may avoid out-of-context operations here and make the code more self-descriptive:
```c++
if (!m_strServerName.IsTrimed())
    m_strServerName = m_strServerName.Trim();
```

Reply:
Done ty!